.github/workflows/main.yml

```diff
@@ -18,21 +18,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18, 20]
-        mongodb-version: [4.4, 5.0, 6.0, 7.0]
+        node-version: [18, 20, 22]
+        mongodb-version: [4.4, 5.0, 6.0, 7.0, 8.0]
```

Comment:
You can remove 4.4 and 5.0, 4.4 is past end of life, 5.0 will be sometime this month if not already